src/freenet/node/PeerNode.java

```diff
@@ -2951,21 +2951,21 @@ public boolean isRoutingBackedOffEither() {
 	long routingBackedOffUntilRT = -1;
 	long routingBackedOffUntilBulk = -1;
 	/** Initial nominal routing backoff length */
-	static final int INITIAL_ROUTING_BACKOFF_LENGTH = (int) SECONDS.toMillis(1);
+	static final int INITIAL_ROUTING_BACKOFF_LENGTH = (int) MILLISECONDS.toMillis(200); // mm and oo: 200ms, toad: 1s
```

Comment:
This is a terrible idea... 200ms is less than the expected RTT... meaning that it ignores the first few backoff rounds